package.json

```diff
@@ -4,13 +4,13 @@
   "description": "Use this to save yourself some time when working on a webpack configuration.",
   "main": "index.js",
   "scripts": {
-    "cover": "nyc --reporter=lcov --reporter=text npm t",
+    "cover": "nyc --reporter=lcov --reporter=text ava --coverage",
```

Comment:
That's odd... `--coverage` doesn't appear to be a flag that `ava` responds to at all... But you say this fixes the problem for you?